WindowsServerDocs/networking/technologies/pktmon/pktmon.md

```diff
@@ -39,21 +39,30 @@ Packet Monitor intercepts packets at multiple locations throughout the networkin
 
 </center>
 
+## Best Practices
+
+Use these best practices for streamlining your network analysis.
+
+-Check command-line help for arguments and capabilities (e.g. pktmon start help).
+-Configure packet filters matching your scenario (pktmon filter add).
+-Check packet counters during the experiment for high level view (pktmon counters).
+-Look into the log for detailed analysis (pktmon format pktmon.etl).
```

Comment:
Looks like you intended to use a bullet point list here. Let's make that list work as intended by adding the required spacing between the list indicators and their following text:
```suggestion
- Check command-line help for arguments and capabilities (e.g. pktmon start help).
- Configure packet filters matching your scenario (pktmon filter add).
- Check packet counters during the experiment for high level view (pktmon counters).
- Look into the log for detailed analysis (pktmon format pktmon.etl).
```

Reply:
By the way, you can test the bullet point list format by adding the same single spaces in your main comment above:

> - removed the mention of release codenames
> - added the best practices section
> - edited the limitations section